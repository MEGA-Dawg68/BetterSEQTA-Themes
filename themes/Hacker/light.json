{
  "css": "https://raw.githubusercontent.com/SethBurkart123/BetterSEQTA-Themes/main/themes/theme.css",
  "className": "",
  "images": {
    "--other-image": "https://source.unsplash.com/random/1920x1080"
  }
}
