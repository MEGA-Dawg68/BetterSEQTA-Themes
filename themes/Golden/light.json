{
  "css": "https://raw.githubusercontent.com/MEGA-Dawg68/BetterSEQTA-Themes/main/themes/theme.css",
  "className": "",
  "images": {
    "--other-image": "https://source.unsplash.com/random/1920x1080"
  }
}
